stubs/protobuf/@tests/struct.py

```diff
@@ -0,0 +1,15 @@
+from google.protobuf.struct_pb2 import ListValue, Struct
```

Comment:
This file is in the wrong place — it needs to be moved to `stubs/protobuf/@tests/test_cases/check_struct.py`, or it won't be tested with mypy, only with pyright